jormungandr-integration-tests/src/common/jormungandr/starter.rs

```diff
@@ -255,18 +255,20 @@ impl Starter {
                 });
             }
             if self.success() {
+                println!("jormungandr is up");
                 return Ok(JormungandrProcess::from_config(
                     process,
                     self.config.clone(),
                 ));
             }
-            if self.stop() {
+            self.custom_errors_found()?;
+            /* if self.stop() {
+                println!("attempt stopped due to error signal recieved");
                 logger.print_raw_log();
                 return Err(StartupError::ErrorInLogsFound {
                     log_content: file_utils::read_file(&self.config.log_file_path),
                 });
-            }
-            self.custom_errors_found()?;
+            }*/
```

Comment:
If this is not meant to be here eventually, just remove the whole thing. We have version control to recover old bits of code.

Reply:
Besides, the method names in `StartupVerification` look like actions, while they are meant to check the outcome. Rename them to something like `check_if_stopped()` for clarity?